src/components/v1/V1Project/modals/ProjectToolDrawerModal.tsx

```diff
@@ -218,6 +259,62 @@ export default function ProjectToolDrawerModal({
             </Form.Item>
           </Form>
         </section>
+
+        <Divider />
+
+        {metadata?.archived ? (
+          <section>
+            <h3>
+              <Trans>Unarchive project</Trans>
+            </h3>
+            <p>
+              <Trans>
+                Your project will immediately appear active on the
+                juicebox.money app. Please allow a few days for it to appear in
+                the "active" projects list on the Projects page.
+              </Trans>
+            </p>
+            <Button
+              onClick={() => setArchived(false)}
+              loading={loadingArchive}
+              size="small"
+              type="primary"
+            >
+              <Trans>Unarchive project</Trans>
+            </Button>
+          </section>
+        ) : (
+          <section>
+            <h3>
+              <Trans>Archive project</Trans>
+            </h3>
+            <p>
+              <Trans>
+                Your project will appear archived, and will not be able to
+                receive payments through the juicebox.money app. You can
+                unarchive a project at any time. Please allow a few days for
+                your project to appear under the "archived" filter on the
+                Projects page.
+              </Trans>
+            </p>
+            <p>
+              <Trans>
+                <strong>Note:</strong>{' '}
+                {terminal?.version === '1.1'
+                  ? 'Unless payments are paused in your funding cycle settings, your project will still be able to receive payments directly through the jbx protocol contracts.'
```

Comment:
**suggestion** can we wrap these strings in `Trans` tags?

Reply:
thank you!!

Reply:
I think there's a way we can lint for missing translations 👀 . Will look into it e v e n t u a l l y